backend/windmill-worker/src/python_executor.rs

```diff
@@ -1493,19 +1489,7 @@ async fn handle_python_deps(
      3. Latest Stable
     */
     let final_version = if is_deployed {
-        // If script is deployed we can try to parse first line to get assigned version
-        if let Some(v) = requirements_lines
-            .get(0)
-            .and_then(|line| PyVersion::parse_version(line))
-        {
-            // We have valid assigned version, we use it
-            v
-        } else {
-            // If there is no assigned version in lockfile we automatically fallback to 3.11
-            // In this case we have dependencies, but no associated python version
-            // This is the case for old deployed scripts
-            PyVersion::Py311
-        }
+        get_pyv_from_requirements(requirements.as_str())
```

Comment:
looks like  a waste if you already have requirement_lines above to do it again in get_pyv_from_requirements non ?